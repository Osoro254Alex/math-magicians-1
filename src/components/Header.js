import React from 'react';
import Navbar from './Navbar';
import Welcome from './Welcome';

class Header extends React.PureComponent {
  render() {
    return (
      <header className="header-container">
        <Navbar />
        <Welcome className="container" />
      </header>
    );
  }
}
export default Header;
