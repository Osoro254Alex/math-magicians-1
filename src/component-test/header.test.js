import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Header from '../components/Header';
import renderer from "react-test-renderer";

afterEach(cleanup);

describe('render header component without error', () => {
  test('header should render correctly', () => {
    render(<Header />);
  });
  test("matches snapshot for new Header", () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
