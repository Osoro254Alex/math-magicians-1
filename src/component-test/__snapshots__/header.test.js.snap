// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render header component without error matches snapshot for new Header 1`] = `
<header
  className="header-container"
>
  <section
    className="heading-section"
  >
    <h1>
      Welcome to our page!
    </h1>
    <p>
      In pariatur elit ipsum sit do velit incididunt ad laborum tempor pariatur esse non. Sit excepteur tempor ipsum nisi elit occaecat sunt proident deserunt ut ullamco. Et pariatur est ipsum ea dolore ullamco consectetur veniam do dolor enim. Commodo tempor ex ipsum laboris ea exercitation cupidatat elit. Labore magna consectetur et voluptate consectetur minim velit mollit.
    </p>
    <p>
      Adipisicing commodo dolor non ipsum eiusmod eiusmod. Lorem amet nisi officia magna mollit eiusmod id adipisicing laboris consequat enim. Tempor irure sunt tempor do enim adipisicing non fugiat duis laborum id mollit. Commodo aliquip duis sit occaecat culpa culpa ea ex officia ullamco consequat. Enim officia esse ad commodo occaecat pariatur ex laboris commodo incididunt sit enim do proident. Reprehenderit ad labore velit cillum do deserunt incididunt ipsum labore laboris in consectetur Lorem. Commodo qui ullamco aliquip minim sunt veniam.
    </p>
  </section>
</header>
`;
